beacon-chain/sync/pending_attestations_queue_test.go

```diff
@@ -173,11 +173,13 @@ func TestProcessPendingAtts_HasBlockSaveAggregatedAtt(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	sig := privKeys[22].Sign(slotRoot[:])
+	// Arbitrary aggregator index for testing purposes.
+	aggregatorIndex := uint64(22)
```

Comment:
I think @prestonvanloon meant to precompute `aggregatorIndex` instead of hardcoding 22. Want me to do it?

Reply:
Sure thing, not sure where 22 came from